VersionControlUI/src/DiffManager.cpp

```diff
@@ -813,6 +843,16 @@ QList<DiffFrame*> DiffManager::createDiffFrames(DiffSetup& diffSetup,
 		auto newNode = const_cast<Model::Node*>(diffSetup.newVersionManager_->nodeIdMap().node(id));
 
 		auto diffNode = new DiffFrame{oldNode, newNode};
+
+		for (auto changeWithNode : changesWithNodes)
```

Comment:
I see you introduced this new argument here `changesWithNodes`. Could you remind me what this was for? Maybe even put a brief comment here before the loop.